Added ability to change color of accent, fixed installer service dialog

J0nathan550 opened the pull request:
Fixes issue: #65 
Allows to change the accent of selected elements in FluentAvalonia.
The item is translated on all languages.

![accent](https://github.com/stayintarkov/SIT.Manager.Avalonia/assets/88881607/a4a06dcc-ad75-48bc-af45-e13a105ca92f)

Plootie commented on SIT.Manager.Avalonia/ViewModels/SettingsPageViewModel.cs:
```diff
@@ -133,4 +141,14 @@ public partial class SettingsPageViewModel : ViewModelBase
+
+    partial void OnCurrentAccentColorChanged(Color? value)
+    {
+        if (faTheme == null) faTheme = Application.Current?.Styles.OfType<FluentAvaloniaTheme>().FirstOrDefault();
```
I've never observed this trigger, Is this needed?

J0nathan550 replied:
> I've never observed this trigger, Is this needed?

For the theme specificly yes, I dig for one hour trying to get the FluentAvaloniaTheme. Lucky me, I found it.

Plootie reviewed (changes requested):
Only request is that you change the binding to `Config.CurrentAccentColor` and apply the accent inside the config changed event similar to how the console font color is handled

J0nathan550 commented:
Refactored. Also fixed one dialog in Installer Service, I was trying to play game with my brother, but noticed that when he had wrong version of Tarkov it pop up him a dialog that would not give him brief dialog that the current version of tarkov is downgraded, plus there was no button at all to close the dialog which locked application forever. Translated it, and fix it.
